### ISSUE TITLE
v3.1.3 is incompatible with graphql-core v3.1.x

### ISSUE THREAD

**Issue by bobbyrenwick** [graphql-python/graphql-relay-py#43]
Using `graphql-core@3.1.4` and `graphql-relay@3.1.3` gives some import errors:

```
2022-01-22 14:12:02,961 P3528 [INFO]        from graphene import UUID, Enum
2022-01-22 14:12:02,961 P3528 [INFO]      File "/var/app/venv/staging-LQM1lest/lib/python3.7/site-packages/graphene/__init__.py", line 2, in <module>
2022-01-22 14:12:02,961 P3528 [INFO]        from .relay import (
2022-01-22 14:12:02,961 P3528 [INFO]      File "/var/app/venv/staging-LQM1lest/lib/python3.7/site-packages/graphene/relay/__init__.py", line 1, in <module
>
2022-01-22 14:12:02,961 P3528 [INFO]        from .node import Node, is_node, GlobalID
2022-01-22 14:12:02,961 P3528 [INFO]      File "/var/app/venv/staging-LQM1lest/lib/python3.7/site-packages/graphene/relay/node.py", line 4, in <module>
2022-01-22 14:12:02,962 P3528 [INFO]        from graphql_relay import from_global_id, to_global_id
2022-01-22 14:12:02,962 P3528 [INFO]      File "/var/app/venv/staging-LQM1lest/lib/python3.7/site-packages/graphql_relay/__init__.py", line 7, in <module>
2022-01-22 14:12:02,962 P3528 [INFO]        from .connection.connection import (
2022-01-22 14:12:02,962 P3528 [INFO]      File "/var/app/venv/staging-LQM1lest/lib/python3.7/site-packages/graphql_relay/connection/connection.py", line 3
, in <module>
2022-01-22 14:12:02,962 P3528 [INFO]        from graphql import (
2022-01-22 14:12:02,962 P3528 [INFO]    ImportError: cannot import name 'GraphQLNamedOutputType' from 'graphql' (/var/app/venv/staging-LQM1lest/lib/python
3.7/site-packages/graphql/__init__.py)
```

This commit added the line that's failing:
https://github.com/graphql-python/graphql-relay-py/commit/bb5f93524e5530fe5eefb2e2bcc6523ec41dc891#diff-b895c3c5263fde224cc02c9d85e98cb6e094c323260e8ea7900db89df72a17a6R13

The [release notes](https://github.com/graphql-python/graphql-relay-py/releases/tag/v3.1.3) say that it should be "compatible with graphql-core version 3.1.".

It seems that's not the case. The exporting of happened in https://github.com/graphql-python/graphql-core/commit/5b9a72888b8b43ee3cd4e14f8583dacbd1bf66b5, which seems to be only available in 3.2.

For now I've worked around this by locking to an earlier version of `graphql-relay-py` and I'm not sure of the exact approach you'd want to use for fixing the problem. Perhaps changing [install_requires in setup.py](https://github.com/graphql-python/graphql-relay-py/blob/6239ab78d155c152ff4c92a57b7e064a959cdc75/setup.py#L35)?

**Comment by Cito** [comment]
Thanks for reporting. You're right, it is unfortunately only compatible with the latest 3.1 version which is 3.1.7. Will try to fix that.

**Comment by Cito** [comment]
Should be fixed now in 3.1.4. Compatibility with graphql-core 3.2 will be provided by graphql-relay-core 3.2.
